testing/releases/releases.go

```diff
@@ -0,0 +1,81 @@
+package releases
```

Comment:
I think these could do with some comments maybe package level otherwise per function.

Perhaps some links elsewhere if we have documented this S3 release 'system'